src/test/java/harness/TestUser.java

```diff
@@ -62,27 +78,97 @@
    * @return global context object, populated with the user's credentials
    */
   public void login() throws IOException {
+    System.out.println("Logging in test user: " + email);
+
     // get domain-wide delegated credentials for this user. use the same scopes that are requested
     // of CLI users when they login.
-    System.out.println("Logging in test user: " + email);
-    GoogleCredentials delegatedUserCredential = getCredentials(User.USER_SCOPES);
+    GoogleCredentials googleCredentials = getCredentials(User.USER_SCOPES);
+
+    // We're not using domain-wide delegation SA key file (for security reasons), so auth is more
```

Comment:
This comment is a little confusing to me since `getCredentials` is still using the domain-wide delegation SA key file to set `.terra/StoredCredential`. It's only `writeBqCredentialFile` and `writeGsUtilCredentialFile` which don't use this.

Reply:
You're right, thanks. Fixed comment and moved.